.github/ISSUE_TEMPLATE/feature_work_template.md

```diff
@@ -20,6 +20,18 @@ _As a [type of user], I want to [do something] so that [this outcome]_
 ### Tasks
 - [ ] _Key pieces of work that will need to be done to meet the User Story and CoAs_
 
+### Definition of Done Checklist
+- [ ] Usability validated
+- [ ] Design QA passed
+- [ ] OESA: Code refactored for clarity
+- [ ] OESA: Dependency rules followed
+- [ ] PR(s) have been merged to main with all the automated tests (load, quality, a11y, unit, etc..) and security scans that are a pre-requisite
```

Comment:
I'm a bit with Brice on this.  I am comfortable with splitting this into two.  Something like...
```markdown
- [ ] PR(s) have been merged to `main`
- [ ] All automated tests (unit, integration, quality, load, a11y, etc...) and security scans pass
```